src/clr-addons/notification/notification.scss

```diff
@@ -78,3 +80,10 @@
     }
   }
 }
+
+@media screen and (max-width: map-get($clr-grid-breakpoints, sm)) {
+  .notification__wrapper {
+    width: calc(100% - 2rem) !important;
```

Comment:
if you place this width definition after the original with 400px, this should overwrite it without having to use !important (as this is bad practice)